rhc-ose-ansible/inventory/ose-provision

```diff
@@ -2,13 +2,14 @@
 
 # Enter the security key to provision new OpenStack instances
 openstack_key_name=""
+#env_id=
 
 # The following variables can be used to override the defaults
 #openshift_master_count=1
 #openshift_node_count=2
 #openshift_storage_disk_volume=/dev/vdb
 #openstack_flavor_name="m1.medium"
-#openstack_image_name="ose3_1-base"
+openstack_image_name="_OS1_rhel-guest-image-7.2-20151102.0.x86_64.qcow2"
 #openshift_openstack_master_storage_size: 10
```

Comment:
these should probably be changed to 
`openshift_openstack_master_storage_size: 10`